codec-dns/src/main/java/io/netty/handler/codec/dns/DefaultDnsRawRecord.java

```diff
@@ -67,12 +67,21 @@ public ByteBuf content() {
 
     @Override
     public DnsRawRecord copy() {
-        return new DefaultDnsRawRecord(name(), type(), dnsClass(), timeToLive(), content().copy());
+        return copy(content().copy());
     }
 
     @Override
     public DnsRawRecord duplicate() {
-        return new DefaultDnsRawRecord(name(), type(), dnsClass(), timeToLive(), content().duplicate());
+        return copy(content().duplicate());
+    }
+
+    @Override
+    public DnsRawRecord rduplicate() {
+        return copy(content().rduplicate());
+    }
+
+    private DnsRawRecord copy(ByteBuf content) {
```

Comment:
`copy` doesn't seem to describe what is being done here. maybe `newDnsRawRecord` or something similar?